resources/benchmark.ts

```diff
@@ -373,11 +373,18 @@ interface BenchmarkSample {
   involuntaryContextSwitches: number;
 }
 
-function sampleModule(modulePath: string): BenchmarkSample {
+function normalizePath(rawPath: string): string {
+  if (process.platform === 'win32') {
+    return path.posix.join(...rawPath.split(path.sep));
+  }
+  return rawPath;
+}
```

Comment:
> normalize benchmark path within sampleCode

Can you please explain the problem that you trying to solve with this code?
Can it be done with https://nodejs.org/api/path.html#pathnormalizepath ?

Reply:
path.normalize would be perfect, thank you!

Reply:
Is this happening because of `localRepoPath()`, if so please add `normalize` call in that specific function?
If not can please explain where non-normalized path is being created?

Reply:
Actually, sorry normalize isn’t exactly what we want. We want to convert a windows style path to a posix style path because even windows wants posix style paths in source code. So we should be able to use path.posix.normalize in the source code

Reply:
fixed with [`url.pathToFileURL`](https://nodejs.org/api/url.html#urlpathtofileurlpath)